netsuite/client.bal

```diff
@@ -318,10 +328,10 @@ public client class Client {
         return getCurrencyResult(response);
     }
 
-    # Gets a currency record from Netsuite by using internal ID
+    # Gets a currency record from Netsuite by using internal ID.
```

Comment:
Change "using internal ID" to "using the internal ID" or "using an internal ID" as relevant.